frontend/src/features/mapSideBar/tabs/propertyDetails/components/LandMeasurementTable.tsx

```diff
@@ -0,0 +1,46 @@
+import { ColumnWithProps, Table } from 'components/Table';
+import { CellProps } from 'react-table';
+import styled from 'styled-components';
+import { formatNumber } from 'utils';
+
+import { TableCaption } from '../../SectionStyles';
+
+export const LandMeasurementTable: React.FC<{ data?: any[] }> = ({ data = [] }) => {
+  return (
+    <>
+      <TableCaption>Land measurement</TableCaption>
+      <StyledTable
+        name="land-measurements"
+        hideHeaders
+        hideToolbar
+        noRowsMessage="No data available"
+        columns={columns}
+        data={data}
+      ></StyledTable>
+    </>
+  );
+};
+
+const columns: ColumnWithProps<any>[] = [
+  {
+    accessor: 'value',
+    align: 'right',
+    Cell: ({ value }: CellProps<any, number>) => formatNumber(value, 0, 2),
+  },
+  { accessor: 'unit', align: 'left' },
+];
+
+const StyledTable = styled(Table)`
```

Comment:
is this something that should be made common?

Reply:
could/should be but this is tailored to the land measurements data so didn't see need for a generic table. Can be refactored at a later date if needed